helpers/deploymentConfig.ts

```diff
@@ -35,13 +35,12 @@ export const preconfiguredAddresses = {
     LzVirtualChainId: "102",
   },
   sepolia: {
-    VTreasury: "0xFc43c055B9be2Ec3BEe6f8C291Af862d764016a0",
+    VTreasury: "0x4116CA92960dF77756aAAc3aFd91361dB657fbF8",
     NormalTimelock: SEPOLIA_MULTISIG,
     FastTrackTimelock: SEPOLIA_MULTISIG,
     CriticalTimelock: SEPOLIA_MULTISIG,
-    AccessControlManager: "0x799700ea540f002134C371fB955e2392FD94DbCD",
-    XVS: "0xD657eB80daA42c334B1c70Cb274E83E4163A3dDc",
-    ResilientOracle: "0x1B85FAbE5c0846662F5FB0E3598fC48eF587e9f0",
+    AccessControlManager: "0xbf705C00578d43B6147ab4eaE04DBBEd1ccCdc96",
```

Comment:
This address (ACM) can be imported from the npm package `@venusprotocol/governance-contracts`, v1.3.0